test/tools/router_check/test/route_tests.sh

```diff
@@ -64,10 +64,25 @@ if [[ "${BAD_CONFIG_OUTPUT}" != *"Unable to parse"* ]]; then
   exit 1
 fi
 
-# Failure test case
-echo "testing failure test case"
+# Failure output flag test cases
+echo "testing failure test cases"
+# Failure test case with only details flag set
 FAILURE_OUTPUT=$("${PATH_BIN}" "${PATH_CONFIG}/TestRoutes.yaml" "${PATH_CONFIG}/Weighted.golden.json" "--details" 2>&1) ||
   echo "${FAILURE_OUTPUT:-no-output}"
-if [[ "${FAILURE_OUTPUT}" != *"expected: [cluster1], actual: [instant-server], test type: cluster_name"* ]]; then
+if [[ "${FAILURE_OUTPUT}" != *"Test_1"*"Test_2"*"expected: [cluster1], actual: [instant-server], test type: cluster_name"* ]]; then
```

Comment:
Is it possible to have a test in which there are multiple errors in the error string vector?

Reply:
^Still working on this by the way, am trying to figure out how to set up the failing test case without failing the earlier tests for expected matches in this file.

Reply:
Sounds good.

/wait

Reply:
Tested for different virtual clusters as a part of the first test, so there are now two errors in Test_2 that get printed.